packages/components/pie-text-input/test/visual/pie-text-input.spec.ts

```diff
@@ -100,7 +100,7 @@ test('Size variants with value and placeholder', async ({ mount, page }) => {
         );
     }));
 
-    await percySnapshot(page, 'PIE Input - Size variants with value and placeholder', percyWidths);
+    await percySnapshot(page, 'Pie Text Input - Size variants with value and placeholder', percyWidths);
```

Comment:
I noticed we use the `Pie` spelling way less than the all caps one. Will it make sense to change only here?

Reply:
Good spot @fernandofranca - I have updated for this file but would suggest we tackle other tests in a separate piece of work as this PR is already quite large

Reply:
Not sure if Github is being misleading but I have indeed changed it to `PIE` - if I understand correctly :) @fernandofranca